cartography/intel/azure/util/credentials.py

```diff
@@ -32,6 +32,9 @@ def __init__(
     def get_current_user(self) -> Optional[str]:
         return self.current_user
 
+    def get_current_subscription_id(self) -> Optional[str]:
```

Comment:
Is this needed? I don't see anyone use this yet. Please confirm if i missed something.